lib/utils/reify-output.js

```diff
@@ -73,10 +78,17 @@ const reifyOutput = arb => {
 // report.  if we get here, we know it's not quiet or json.
 const printAuditReport = arb => {
   const reporter = npm.command !== 'audit' ? 'install' : 'detail'
-  const res = auditReport(arb.auditReport, {
-    reporter,
-    ...npm.flatOptions
-  })
+  let res
+  if (arb.auditReport) {
+    res = auditReport(arb.auditReport, {
+      reporter,
+      ...npm.flatOptions
+    })
+  } else {
+    res = {
+      report: 'No audit report found'
```

Comment:
Probably the better thing to do here is just return.  If you run `npm install --no-audit` and then get a notice that no audit report was found, it's just noise.